src/main/java/com/google/devtools/build/lib/actions/ActionOwner.java

```diff
@@ -39,14 +39,14 @@ public abstract class ActionOwner {
   @SerializationConstant
   public static final ActionOwner SYSTEM_ACTION_OWNER =
       createDummy(
-          /* label= */ null,
+          /* label= */ Label.parseCanonicalUnchecked("@_builtins//virtual:system_action_owner"),
```

Comment:
Makes sense, although I've a minor concern that having a vestigial (unused) mirror of this platform could lead to behavioral differences if used at some later date. Since there would technically be 2 separate definitions (one in Java, another in Starlark).